Presentation.Web/app/Configs/paragraphSizeConfig.ts

```diff
@@ -0,0 +1,7 @@
+﻿module Kitos.Configs {
+
+    export class ParagraphSizeConfig {
```

Comment:
Det er vel ret specifikt for relationstabellen. Den her kan ikke rigtig genbruges til andet, så måske den skulle hedde noget der var lidt mere specifikt. I bør desuden tjekke at forholder 199 og 4 stadig er korrekt, da de blev sat dengang der var mere bredde på dén kolonne. Jeg forestiller mig heller ikke at det er samme config i kontraktoverblikket, hvorfor denne konfiguration faktisk ikke er generel?